pkg/files/files_test.go

```diff
@@ -109,7 +109,13 @@ func TestGetRelativePath(t *testing.T) {
 		t.Skip("Windows fails if current directory is deleted")
 	}
 
-	DIR := "/tmp/stuff"
+	var DIR string
+	expectedPath := "../.."
+	if runtime.GOOS == "darwin" {
+		DIR = "/private"
+		expectedPath += "/.."
+	}
+	DIR += "/tmp/stuff"
```

Comment:
Could we use `os.createTempDir` instead of handling finding the correct path on our own?
https://pkg.go.dev/os#CreateTemp